contractsV5/KyberNetwork.sol

```diff
@@ -679,6 +679,74 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
         // create array of reserves receiving fees + fee percent per reserve
         // fees should add up to 100%.
         // send total fee amount to fee handler with reserve data.
+
+        address[] memory eligibleReserves = new address[](tradeData.numFeePayingReserves);
+        uint[] memory rebatePercentages = new uint[](tradeData.numFeePayingReserves);
+
+        // Updates reserve eligibility and rebate percentages
+        updateEligibilityAndRebates(eligibleReserves, rebatePercentages, tradeData);
+
+        // Sending platform fee to taker platform
+        (bool success, ) = tradeData.input.platformWallet.call.value(tradeData.platformFeeWei)("");
+        require(success, "Transfer platform fee to taker platform failed");
+
+        // Send total fee amount to fee handler with reserve data.
```

Comment:
we want to send the rebate wallets.
not the reserve address

Reply:
Yes but we agreed I was not going to optimise it for rebate wallets first and just use reserve address.

Reply:
Sorry misunderstanding, I've fixed it.

Reply:
sure

I idea when writing non optimized code is we do the correct flow. we might deploy if.
or later if we have time we optimize it. or we have any other good reason

no point in sending to wrong address.